.github/workflows/weekly-cron-test.yml

```diff
@@ -2,7 +2,7 @@ name: Weekly CNI tests
 
 on:
   schedule:
-    - cron: "0 16 * * 2"  # every Tuesday
+    - cron: "0 22 * * 3"  # every Wednesday
```

Comment:
Curious, why this change?

Reply:
We would want to execute all steps in weekly perf test despite failures of any one step in the workflow. Followed suggestion in https://github.community/t/continue-on-error-allow-failure-ui-indication/16773/2 to achieve this functionality. 

Change in cron schedule here is to test the effect of this workflow change today.